src/main/java/SYSC4806/Model/Course.java

```diff
@@ -19,7 +16,11 @@
     private int year;
     private String name;
     private String code;
+
+    @ManyToMany
     private ArrayList<LearningOutcome> learningOutcomes;
+
+    @ManyToMany
```

Comment:
Referencing https://www.baeldung.com/spring-data-rest-relationships, I think this needs to have a CascadeType.ALL and it looks like they use @JoinTable to create that intermediary table for many to many relationships.